backend/fregepoc/repositories/tasks.py

```diff
@@ -104,6 +107,8 @@ def process_repo_task(repo_pk):
 
     for repo_file in repo_files:
         analyze_file_task.apply_async(args=(repo_file.pk,))
+
+    repo_obj.close()
     _finalize_repo_analysis(repo)
 
 
```

Comment:
Please, remove redundant empty lines at the end of the files. Have you already installed `pre-commit` hooks? If you install them, they will be doing this kind of stuff for you.